docs/en/action-menu-items/custom-browser-protocols.md

```diff
@@ -11,10 +11,10 @@ lang: en
 
 - [Registering a protocol](#registering-a-protocol)
   - [Registering a protocol on Windows](#registering-a-protocol-on-windows)
-  - [Registering a protocol on OSX](#registering-a-protocol-on-osx)
+  - [Registering a protocol on macOS](#registering-a-protocol-on-macOS)
   - [Registering a protocol on Linux](#registering-a-protocol-on-linux)
 
-A very practical version of an [Action Menu Item]() (AMI) is a variant that will run an application or a script on your local computer. For this to work you need to set up a connection between your browser and the script or application you wish to run. This link is called a custom browser protocol.
+A very practical version of an [Action Menu Item](https://developer.shotgridsoftware.com/67695b40/) (AMI) is a variant that will run an application or a script on your local computer. For this to work you need to set up a connection between your browser and the script or application you wish to run. This link is called a custom browser protocol.
```

Comment:
Can you remove the host part of this link since the source and destination are on the same host/repo?

Reply:
Sure thing and thank you for pointing this out!

Reply:
Fixed!